### DISCUSSION TITLE
pyAFQ with stroke lesions

### DISCUSSION THREAD

**Discussion by arokem** [yeatmanlab/pyAFQ#716] [General]
Moving this to a discussion from #715 

Dear pyAFQ team,

Please, there's anny recomendation or citation showing the possibility to use pyAFQ with Stroke Brain Lesions?

I ran the pyAFQ code integrated with AFQ-Browser and, obviously, the tractography is not showing some tracts near the lesion. But I was wondering if it's possible to adjust the pyAFQ parameters to get a better tractography with stroke patients? Maybe reduce the fa threshold?
Sorry for this beginner's tractography question.

Thank you!

**Comment by arokem** [comment]
Thanks for asking this question @dubergonzoni! 

The quick answer is that we haven't done this ourselves, so there is no immediate citation to show that this software works well for that use-case. But we are interested in this topic. And yes: tractography within stroke lesions is challenging... One way to potentially deal with these challenges is to use a diffusion modeling method that deals with the edema in and around the lesion. We have a work-in-progress pull request to integrate such a method into pyAFQ https://github.com/yeatmanlab/pyAFQ/pull/672 (it's base on a method by Hoy et al. that we described in [this paper](http://rescience.github.io/bibliography/henriques_2017.html)). This will require multi b-value data, but other methods are [currently under development in DIPY](https://github.com/dipy/dipy/pull/2087) that would remove this constraint.

Another is to use a bundle recognition algorithm that deals well with displacements of the bundles. The standard waypoint ROI method is somewhat robust to this, but it might be worth also trying the "recobundles" bundle recognition option. The [recobundles paper](https://pubmed.ncbi.nlm.nih.gov/28712994/) has a couple of examples of recognition of bundles even with tumors displacing the bundle.

**Reply by dubergonzoni** [reply]
Thank you so much for your quickly reply @arokem!

I ran recobundles with my data and it looks great. I will study and follow up the developing of the DIPY method.

One last question and, please, answer if you can: 

I have used the 30 bundles from RecoBundles. To use the Callosum bundles, I have to create the .trk files like this DIPY tutorial? https://yeatmanlab.github.io/pyAFQ/auto_examples/plot_callosal_tract_profile.html

And to use the Pandora Atlas, can I convert .vtk to .trk files with DIPY? I have tried with dipy.io.streamline.load_vtk but I got an empty list.

Thank you Dr. Roken.

**Reply by dubergonzoni** [reply]
Regarding the Callosum Bundles, it's possible to obtain form Dipy team the .trk files from health subjects created during the development of this Calossum Bundles? Maybe @jyeatman have this answer? I'm interested in the PosteriorParietal, SupParietal and Temporal .trk files to use with RecoBundles.
I'm trying, but I could not create myself the specific .trk files adapting the tutorial "plot_callosal_tract_profile".

Thank you for your patience and sorry about that amount of questions.
